scripts/env-setup.sh

```
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Load environment variables from .env file if it exists
+if [ -f .env ]; then
+    set -a # If all variables that are defined separately are exported to the environment of the subsequent commands
+    source .env
+    set +a # Turn of alexport
+fi
+
+# git related commands
+export GITHUB_SHA=$(git rev-parse --short HEAD)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)